app/js/ui.js

```diff
@@ -1,22 +1,21 @@
+//Get csv from user upload
 document.getElementById('upload_link').onclick = function() {
     document.getElementById('input_csv').click();
     userUploadFlag = true;
 };
 
-/*
- * Code to run when document is ready
- */
+//Modal settings, as defined by Material Design Guidelines
```

Comment:
Material Design Guidelines doesn't define CSS as guidelines. This should be rephrased as "...., as referenced in Materialize CSS"